logging/resource.go

```diff
@@ -126,15 +126,23 @@ func detectCloudFunction() *mrpb.MonitoredResource {
 	}
 }
 
-func (r *resource) isCloudRun() bool {
+func (r *resource) isCloudRunService() bool {
```

Comment:
This seems to be a public function, right? If so, we shouldn't break the signature unless absolutely necessary

Reply:
I thought in go only functions with names starting with capital letters were public? I'm guessing I either have that wrong (not very familiar with the language) or am misunderstanding what you mean by "public."

Assuming I revert this, are you saying I should keep isCloudRun() alone and have it only identify services and add isCloudRunJob() as a new method identifying only jobs?

Reply:
> This seems to be a public function, right? If so, we shouldn't break the signature unless absolutely necessary

I don't think this is a public function. Additionally, these boolean functions that check if a resource is an App Engine/Cloud Run/Cloud Function resource are exclusively used within `resource.go` for the `detectResource` function.